go.mod

```diff
@@ -0,0 +1,54 @@
+module github.com/openshift/aws-account-operator
+
+go 1.13
+
+require (
+	github.com/aws/aws-sdk-go v1.19.49
+	github.com/evanphx/json-patch v4.5.0+incompatible // indirect
+	github.com/go-logr/logr v0.1.0
+	github.com/go-openapi/spec v0.19.0
+	github.com/golang/mock v1.3.1-0.20190508161146-9fa652df1129
+	github.com/hashicorp/golang-lru v0.5.4 // indirect
+	github.com/onsi/ginkgo v1.14.1
+	github.com/onsi/gomega v1.10.2
+	github.com/openshift/api v3.9.1-0.20190424152011-77b8897ec79a+incompatible
+	github.com/openshift/operator-custom-metrics v0.3.0
+	github.com/operator-framework/operator-sdk v0.12.0
+	github.com/prometheus/client_golang v1.1.0
+	github.com/spf13/pflag v1.0.3
+	github.com/stretchr/testify v1.4.0
+	go.uber.org/atomic v1.4.0 // indirect
+	go.uber.org/zap v1.10.0 // indirect
+	golang.org/x/net v0.0.0-20200904194848-62affa334b73 // indirect
+	golang.org/x/sys v0.0.0-20200908134130-d2e65c121b96 // indirect
+	golang.org/x/text v0.3.3 // indirect
+	golang.org/x/xerrors v0.0.0-20200804184101-5ec99f83aff1 // indirect
+	gopkg.in/yaml.v2 v2.3.0
+	k8s.io/api v0.0.0
+	k8s.io/apimachinery v0.0.0
+	k8s.io/client-go v11.0.0+incompatible
+	k8s.io/klog v0.3.3 // indirect
+	k8s.io/kube-openapi v0.0.0-20190320154901-5e45bb682580
+	sigs.k8s.io/controller-runtime v0.3.0
+	sigs.k8s.io/yaml v1.2.0 // indirect
+)
+
+replace gopkg.in/fsnotify.v1 v1.4.9 => github.com/fsnotify/fsnotify v1.4.9
+
+// Pinned to kubernetes-1.13.1
+replace (
+	k8s.io/api => k8s.io/api v0.0.0-20181213150558-05914d821849
+	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.0.0-20181213153335-0fe22c71c476
+	k8s.io/apimachinery => k8s.io/apimachinery v0.0.0-20181127025237-2b1284ed4c93
+	k8s.io/client-go => k8s.io/client-go v0.0.0-20181213151034-8d9ed539ba31
+)
+
+replace (
+	github.com/coreos/prometheus-operator => github.com/coreos/prometheus-operator v0.29.0
+	k8s.io/code-generator => k8s.io/code-generator v0.0.0-20181117043124-c2090bec4d9b
+	k8s.io/kube-openapi => k8s.io/kube-openapi v0.0.0-20180711000925-0cf8f7e6ed1d
+	sigs.k8s.io/controller-runtime => sigs.k8s.io/controller-runtime v0.1.10
+	sigs.k8s.io/controller-tools => sigs.k8s.io/controller-tools v0.1.11-0.20190411181648-9d55346c2bde
+)
+
+replace github.com/operator-framework/operator-sdk => github.com/operator-framework/operator-sdk v0.9.0
```

Comment:
...this equals weird. Why wouldn't we just use `v0.9.0` on L16?

Reply:
Yeah, this was weird to me as well.  It was in the documentation to add the items below `// Pinned to kubernetes-1.13.1` coming up from v0.8.2, which prevented conflicts with the k8s client-runtime, which from what I understand, stems from an ideological argument between K8S maintainers and the Go maintainers over semantic versioning.